keyboards/pos78/config.h

```diff
@@ -0,0 +1,110 @@
+/*
+Copyright 2020 smssmssms
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID    0x076c
+#define PRODUCT_ID   0x0000
```

Comment:
As mentioned by @drashna this should be a unique VID/PID combination. Generally try and avoid using IDs that are known to be in use. There's search engines like the one drashna linked.

There's a little blurp about IDs in the docs: https://docs.qmk.fm/#/faq_build?id=usb-vid-and-pid
as for ensuring there's no PID overlap, you can use this tool to check :) https://yanfali.github.io/qmk_usb_usage/

Reply:
OK, think I've sorted it.

Reply:
Used 0xFEED and 0x7878